src/foundry/prosemirror/menu.d.mts

```diff
@@ -203,4 +220,24 @@ declare class ProseMirrorMenu extends ProseMirrorPlugin {
    * Clear any marks from the current selection.
    */
   protected _clearFormatting(): void;
+
+  /**
+   * Toggle link recommendations
+   */
+  protected _toggleMatches(): Promise<void>;
+
+  /**
+   * Toggle the given selection by wrapping it in a given block or lifting it out of one.
+   * @param node  - The type of node being interacted with.
+   * @param wrap  - The wrap command specific to the given node.
+   * @param attrs - Attributes for the node.
```

Comment:
Does this really work for documenting `attrs`? The last time I tried this, I don't think it did. You would have to place the documentation on the type instead.

Reply:
Will adjust to `options?: { attrs?: object }` and move the type documentation for `attrs` inside